package.json

```diff
@@ -30,6 +30,7 @@
   },
   "devDependencies": {
     "@cdwr/nx-payload": "0.11.0",
```

Comment:
Must reference a built package, either in e.g. `/node_modules` or `/dist`.
We don't have `/dist` at start and it's not possible to run `pnpm i` and build since `nx.json` can't resolve the plugin.

So, it's a "moment 22".